tests/rtp-mode.rs

```diff
@@ -0,0 +1,140 @@
+use std::collections::VecDeque;
+use std::net::Ipv4Addr;
+use std::time::Duration;
+use tracing::info_span;
+
+use str0m::format::Codec;
+use str0m::media::rtp::RtpHeader;
+use str0m::media::rtp::{Extension, ExtensionMap};
+use str0m::media::{Direction, MediaKind};
+use str0m::{Candidate, Event, Rtc, RtcError};
+
+mod common;
+use common::{init_log, progress, TestRtc};
+
+#[test]
+pub fn rtp_mode() -> Result<(), RtcError> {
+    init_log();
+
+    let rtc1 = Rtc::builder().set_rtp_mode(true).build();
+    let rtc2 = Rtc::builder()
+        .set_rtp_mode(true)
+        // release packet straight away
+        .set_reordering_size_audio(0)
+        .build();
+
+    let mut l = TestRtc::new_with_rtc(info_span!("L"), rtc1);
+    let mut r = TestRtc::new_with_rtc(info_span!("R"), rtc2);
+
+    let host1 = Candidate::host((Ipv4Addr::new(1, 1, 1, 1), 1000).into())?;
+    let host2 = Candidate::host((Ipv4Addr::new(2, 2, 2, 2), 2000).into())?;
+    l.add_local_candidate(host1);
+    r.add_local_candidate(host2);
+
+    let mut change = l.sdp_api();
+    let mid = change.add_media(MediaKind::Audio, Direction::SendRecv, None, None);
+    let (offer, pending) = change.apply().unwrap();
+
+    let answer = r.rtc.sdp_api().accept_offer(offer)?;
+    l.rtc.sdp_api().accept_answer(pending, answer)?;
+
+    loop {
+        if l.is_connected() || r.is_connected() {
+            break;
+        }
+        progress(&mut l, &mut r)?;
+    }
+
+    let max = l.last.max(r.last);
+    l.last = max;
+    r.last = max;
+
+    let params = l.media(mid).unwrap().payload_params()[0];
+    assert_eq!(params.spec().codec, Codec::Opus);
+    let pt = params.pt();
+
+    let mut exts = ExtensionMap::empty();
+    exts.set(3, Extension::AudioLevel);
+
+    let to_write = vec![
```

Comment:
using RtpHeader here to construct the header would make the test easer to read and to create more tests.